src/hooks/useVerifiedLocations.tsx

```diff
@@ -200,6 +202,23 @@ export const transformBabalaResponse: RT<BabalaResponse, BabalaData> = (
   };
 };
 
+export const transformTeyitEnkazResponse: RT<
+  TeyitEnkazResponse,
+  TeyitEnkazData
+> = (res) => {
+  return {
+    channel: "teyit_enkaz",
+    geometry: createGeometry(res),
+    properties: {
+      name: res.extraParams?.name ?? null,
+      description: res.extraParams?.description ?? null,
+      type: res.extraParams?.styleUrl ?? null,
+      icon: null, // TODO: fix this after we have an icon
```

Comment:
@oguzergul once we have the image file for this, we can change this line to `icon: res.extraParams?.icon ?? null` to use the icon path from response. But i am not sure where we will find the icon from. How did you do it for the previous ones?